ShowBuildMenu.bat

```diff
@@ -1,12 +1,16 @@
 @echo off
 pushd %~dp0
 
+for /f "tokens=* USEBACKQ delims= " %%i in (`findstr /c:"NUnit.ConsoleRunner" "Tools\packages.csproj"`) do set NUNIT_VERSION=%%i
+set NUNIT_VERSION=%NUNIT_VERSION:~57%
+set NUNIT_VERSION=%NUNIT_VERSION:" />=%
+
 set NANT="%~dp0Tools\nant\bin\NAnt.exe" -t:net-4.0
 set BUILD_TOOL_PATH=%~dp0Tools\BuildTool\bin\BuildTool.dll
 set BUILDTOOL=dotnet %BUILD_TOOL_PATH%
 set AVAILABLE_CONFIGURATIONS=%~dp0available-test-configurations
 set CURRENT_CONFIGURATION=%~dp0current-test-configuration
-set NUNIT="%~dp0Tools\NUnit.ConsoleRunner.3.7.0\tools\nunit3-console.exe"
+set NUNIT="%userprofile%\.nuget\packages\NUnit.ConsoleRunner\%NUNIT_VERSION%\tools\nunit3-console.exe"
```

Comment:
Wouldn't it be better to keep those tools inside Tools folder?
I think it should be possible by adding "nuget.config" file to Tools folder with the following content:
```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <config>
    <add key="repositoryPath" value="./packages" />
  </config>
</configuration>
```

Reply:
`nuget.config` didn't work for `dotnet restore`, but it worked with `--packages` option. Now the packages are restored into Tools folder as before.